sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -618,3 +619,45 @@ case class SQLKeywords() extends LeafExpression with Generator with CodegenFallb
 
   override def prettyName: String = "sql_keywords"
 }
+
+@ExpressionDescription(
+  usage = """_FUNC_() - Get all of the Spark SQL string collations""",
+  examples = """
+    Examples:
+      > SELECT * FROM _FUNC_() LIMIT 2;
+       SYSTEM  BUILTIN  UTF8_BINARY NULL  NULL  ACCENT_SENSITIVE  CASE_SENSITIVE  NO_PAD  NULL
+       SYSTEM  BUILTIN  UTF8_LCASE  NULL  NULL  ACCENT_SENSITIVE  CASE_INSENSITIVE  NO_PAD  NULL
```

Comment:
Is this output deterministic?

Reply:
Yes, `UTF8_BINARY` and `UTF8_LCASE` have always been at the forefront
<img width="762" alt="image" src="https://github.com/user-attachments/assets/0f2bc20d-e51d-4f14-b51e-61046d3315ef">

Reply:
It should be, listCollationMeta enforces UTF8_* collations first.

Reply:
I would replace this `LIMIT` by `WHERE` which always returns one row, to don't depend on order and how LIMIT behaves.

Reply:
Updated.